api/src/main/java/jakarta/persistence/AssociationOverride.java

```diff
@@ -29,46 +29,44 @@
  * embeddable within an entity to another entity when the embeddable
  * is on the owning side of the relationship. When used to override a
  * relationship mapping defined by an embeddable class (including an
  * embeddable class embedded within another embeddable class),
- * <code>AssociationOverride</code> is applied to the field or
- * property containing the embeddable.
+ * {@code AssociationOverride} is applied to the field or property
+ * containing the embeddable.
  *
- * <p> When <code>AssociationOverride</code> is used to override a
- * relationship mapping from an embeddable class, the
- * <code>name</code> element specifies the referencing relationship
- * field or property within the embeddable class. To override mappings
- * at multiple levels of embedding, a dot (".") notation syntax must
- * be used in the <code>name</code> element to indicate an attribute
- * within an embedded attribute.  The value of each identifier used
- * with the dot notation is the name of the respective embedded field
- * or property.
+ * <p> When {@code AssociationOverride} is used to override a
+ * relationship mapping from an embeddable class, the {@link #name}
+ * element specifies the referencing relationship field or property
+ * within the embeddable class. To override mappings at multiple
+ * levels of embedding, a dot {@code .} notation syntax must be used
+ * in the {@code name} element to indicate an attribute within an
+ * embedded attribute. The value of each identifier used with the dot
+ * notation is the name of the respective embedded field or property.
  * 
- * <p> When <code>AssociationOverride</code> is applied to override
- * the mappings of an embeddable class used as a map value,
- * "<code>value.</code>" must be used to prefix the name of the
- * attribute within the embeddable class that is being overridden in
- * order to specify it as part of the map value.
+ * <p>When {@code AssociationOverride} is applied to override the
+ * mappings of an embeddable class used as a map value, "{@code value.}"
+ * must be used to prefix the name of the attribute within the embeddable
+ * class that is being overridden in order to specify it as part of the
+ * map value.
  *
- * <p> If the relationship mapping is a foreign key mapping, the
- * <code>joinColumns</code> element is used.  If the relationship
- * mapping uses a join table, the <code>joinTable</code> element must
- * be specified to override the mapping of the join table and/or its
- * join columns.
+ * <p>If the relationship mapping is a foreign key mapping, the
+ * {@link #joinColumns} element is used. If the relationship mapping
+ * uses a join table, the {@link #joinTable} element must be specified
+ * to override the mapping of the join table and/or its join columns.
  *
+ * <p>Example 1: Overriding the mapping of a relationship defined by a
+ * mapped superclass
  * <pre>
- *    Example 1: Overriding the mapping of a relationship defined by a mapped superclass
```

Comment:
I was to say that this should have been updated to use `<pre>{@code ...` but then I realised #508 is a better way

Reply:
Yeah, we need @snippet. Actually I find that `<pre>{@code` doesn't work well.